src/index.js

```diff
@@ -21,6 +21,12 @@ export class Animate extends React.Component {
     this.animating = false;
   }
 
+  componentWillMount() {
+    if (this.props.updateOnMount) {
```

Comment:
Gotcha. In my case this was a little awkward because the class which generates the translateX value isn't the same class which generates the thing being animated (the parent generates the thing). I solved it by cloning the element and adding the starting props. From my render method:

```js
if (!this.initialStylestyleAttrs) {
  this.initialStylestyleAttrs = {
    opacity: this.state.opacity,
    transform: `translateX(${this.state.offsetPx}px)`,
  }
  if (this.props.autoWidth) {
    this.initialStylestyleAttrs.width = `${this.state.scalePx}px`;
  }
};

const extendedChild = React.cloneElement(this.props.children, {
  style: this.initialStylestyleAttrs,
});

return (
  <Animate
    translateX={ this.state.offsetPx }
    opacity={ this.state.opacity }
    tension={ this.props.tension }
    friction={ this.props.friction }
    ref="item"
    animate={ this.props.animate }
  >
    { extendedChild }
  </Animate>
);
```

This appears to work without the `updateOnMount` prop, so I'm going to drop that commit from my pull request.